Automattic/jetpack
# Add basic E2E WAF tests

Add some basic WAF tests to check if requests are blocked when they should.

#### Changes proposed in this Pull Request:

* Add the spec for WAF e2e test
* Add simple test for request blocking

#### Other information:


#### Jetpack product discussion



While detailed tests of the WAF engine are better suited to unit tests, and also need access to the full set of rules which are not present here, we still believe a single E2E test to ensure that requests are correctly blocked when the module is enabled is valuable.

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* See CI checks
* Run locally with `$ pnpm test:run specs/waf/blocking.test.js --headed` from within `projects/plugins/jetpack/tests/e2e`
